src/templates/addon/list.html

```diff
@@ -0,0 +1,33 @@
+{% include '_macros/addon_tile.html' %}
+
+<section class="main">
+  <div class="subheader">
+    <h1>{{ _('Firefox OS Add-ons') }}</h1>
+  </div>
+</section>
+
+<section class="main full c">
+  {% defer (url=api('addon_list'), pluck='objects', as=addons) %}
```

Comment:
I forget, too. Was hoping to get this merged and iterate if it didn't work once the endpoint is working.